src/app/battlefield/battlefield.component.ts

```diff
@@ -15,11 +15,11 @@ export class BattlefieldComponent implements OnInit {
   battlefield?: Battlefield;
   targetables?: Targetable[];
 
-  constructor(private battlefieldService: BattlefieldService) {}
+  constructor(private combatState: CombatStateService) {}
```

Comment:
you don't need to dependency-inject this, just import `Game` and use it directly.